src/app/app.component.spec.ts

```diff
@@ -57,4 +100,42 @@ describe('AppComponent', () => {
     fix.detectChanges();
     expect(de.query(By.css('prx-navuser'))).toBeNull();
   }));
+
+  it('should load series podcast and episode and call CMS actions', () => {
+    spyOn(CmsActions, 'cmsPodcastFeed').and.callThrough();
+    spyOn(CmsActions, 'cmsEpisodeGuid').and.callThrough();
+    const podcast: PodcastModel = {
+      doc: auth.mock('prx:series', {seriesId: 37800, title: 'Pet Talks Daily'}),
+      seriesId: 37800,
+      feederId: '70',
+      title: 'Pet Talks Daily'
+    };
+    const filter: FilterModel = {
+      podcast,
+      beginDate: new Date('2017-08-27T00:00:00Z'),
+      endDate: new Date('2017-09-07T00:00:00Z'),
+      interval: INTERVAL_DAILY
+    };
+    const series = auth.mockItems('prx:series', [
+      {seriesId: 37800, title: 'Pet Talks Daily'},
+      {seriesId: 37801, title: 'Totally Not Pet Talks Daily'}]);
+    series.forEach(s => {
+      const distributions = s.mockItems('prx:distributions', [{kind: 'podcast', url: 'https://feeder.prx.org/api/v1/podcasts/70'}]);
+      const episodes = s.mockItems('prx:stories', [{id: 123, title: 'A Pet Talk Episode', publishedAt: new Date()}]);
+      episodes.forEach(e => {
+        e.mockItems('prx:distributions',
+          [{kind: 'episode', url: 'https://feeder.prx.org/api/v1/episodes/42b4ad11-36bd-4f3a-9e92-0de8ad43a515'}]);
+      });
+    });
+    // TODO this is an awful lot of what looks like redundant mocking. will try to do a better set up but just trying to get there for now
```

Comment:
This is a mess, but because of how I stored those haldocs with the entries in the store, I was getting unmocked request from both the filter store and the podcast store. I will be looking at how I might be able to clean this up in the tests and maybe rethink how I'm accessing the haldocs to make requests.